test/integration/test-fixtures/image-large-apk-data/Dockerfile

```diff
@@ -1,5 +1,2 @@
 FROM alpine@sha256:d9a7354e3845ea8466bb00b22224d9116b183e594527fb5b6c3d30bc01a20378
-RUN apk add --no-cache \
-            tzdata=2021a-r0 \
-            vim=8.2.2320-r0 \
-            alpine-sdk=1.0-r0
+RUN apk add --no-cache tzdata vim alpine-sdk
```

Comment:
We pin to dependency versions intentionally for reproducibility and to avoid surprise test failures.

What was the exact issue caused by those pinned versions? Perhaps there are newer versions that can be pinned to solve things?